kumascript/macros/IncludeSubnav.ejs

```diff
@@ -1,14 +0,0 @@
-<%
-// Inserts the contents of a sub navigation into the document.
-//
-// Parameters:
-//
-//  $0 - URL of a page with a "Subnav" section to insert.
-//       Example: "/en-US/docs/Games".
-//       Do not use https://etc.
-//  
-
-var url = web.spacesToUnderscores($0);
-
-%>
-<%- await wiki.page(url, 'Subnav', '', 1) %>
```

Comment:
I don't get it. Why is this deleted? 
It's not used by any en-US docs, but it's used in hundreds of pages in the translated content. 
If you delete the macro, won't that cause a bunch of `macros` flaws and the kumascript renderer will display something like `<code>{{IncludeSubna('bla/bla')}}</code>` in the final HTML.

Reply:
Having said that, I know `translated-content/files/kab/mdn/contribute/index.html` uses the macro but when I view it, it's not throwing a flaw error or rendering the raw KS source in its place. 
Actually nothing's mentioned about that busted macro call when I view http://localhost:3000/kab/docs/MDN/Contribute/
I don't get it.

Reply:
That's because I did not remove the file. Just the content. So it still executes fine.

I think it never renders successfully and that's why I removed it. But I'm restoring it and we deal with it later. Sorry for the confusion.

Reply:
I got tricked by the GitHub PR diff view!
What it should do is instead of restoring it is to change it to be this:
```
<%
// The IncludeSubnav macro is considered deprecated. 
// As of today, Feb 2021, no content in the en-US repo uses it. 
// The few translated-content files that were using it were buggy and 
// we decided it wasn't worth even trying to recover that because no
// content should be using it anyway.
mdn.deprecated();
%>
```

I haven't tested it but I think it does the same output (empty string) and now there it becomes a fixable flaw. The flaw is that it's used, and the fix it to just remove the whole `{{IncludeSubnav(.*?)}}` which I think is supported already.